tests/unit/phpunit.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<phpunit bootstrap="bootstrap.php"
+		 verbose="true"
+		 timeoutForSmallTests="900"
+		 timeoutForMediumTests="900"
+		 timeoutForLargeTests="900"
+>
+	<testsuite name='ownCloud - Template Editor App Tests'>
+		<directory suffix='Test.php'>.</directory>
+	</testsuite>
+	<!-- filters for code coverage -->
+	<filter>
+		<whitelist>
+			<directory suffix=".php">../../../templateeditor</directory>
+			<exclude>
+				<directory suffix=".php">../../../templateeditor/l10n</directory>
+				<directory suffix=".php">../../../templateeditor/tests</directory>
```

Comment:
weird, I thought it would be enough to just put `..` here since that would point at the tests folder, based on the path where phpunit.xml is located ?

Reply:
Damn... so other apps are doing that too.

Can you adjust so that to avoid next copy-paste victim ? :-D